src/Components/StepTwo/StepTwo.js

```diff
@@ -1,8 +1,118 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import './StepTwo.scss';
+import axios from 'axios';
+import useFetchForm from '../useFetchForm/useFetchForm';
+import { url2 } from '../../Constants';
 
+const StepTwo = ({prevStep, nextStep, data, setData}) => {
+
+  const {dataStates} = useFetchForm(url2);
+  
+  const [errors, setErrors] = useState({})
+  const [dataCities, setDataCities] = useState([])
+  const [dataPostalCodes, setDataPostalCodes] = useState([])
+  const [tla, setTla] = useState('');
+    
+  useEffect(()=> {
+        if (tla) {
+        axios.get(url2 + "/" + tla + "/cities")
+        .then(res => {
+            setDataCities(res.data)
+        })
+    }
+    },[tla])
+
+    const [inputs, setInputs] = useState({
```

Comment:
Also good place to use useReducer :)

Reply:
I fixed other states putting them into one big state. This Step 2 is little bit complicated because I am bringing all the data there for multiple reasons. I am willing to try use Reducer but since I have never used it before, I was thinking to complete the project with regular states and when I all done, to make another branch and practice implementing use Reducer. Of course if you are ok with that? :)